packages/react/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "@parallelmarkets/react",
+  "version": "0.0.1",
+  "description": "ParallelMarkets.com React SDK",
+  "author": "Parallel Markets (https://parallelmarkets.com)",
+  "license": "MIT",
+  "homepage": "https://developer.parallelmarkets.com/docs/javascript",
+  "main": "dist/parallelmarkets-react.cjs.js",
+  "module": "dist/parallelmarkets-react.esm.js",
+  "scripts": {
+    "build": "NODE_ENV=production rollup -c",
+    "lint": "NODE_ENV=test eslint ./src && prettier -l ./src"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.15.0",
+    "@babel/eslint-parser": "^7.15.0",
+    "@babel/plugin-proposal-class-properties": "^7.14.5",
+    "@babel/plugin-proposal-private-methods": "^7.14.5",
+    "@babel/plugin-proposal-private-property-in-object": "^7.14.5",
+    "@babel/plugin-syntax-jsx": "^7.14.5",
+    "@babel/preset-env": "^7.15.0",
+    "@rollup/plugin-babel": "^5.3.0",
+    "@rollup/plugin-commonjs": "^20.0.0",
+    "@rollup/plugin-node-resolve": "^13.0.4",
+    "babel-preset-react-app": "^10.0.0",
+    "eslint": "^7.32.0",
+    "eslint-config-prettier": "^8.3.0",
+    "eslint-plugin-import": "^2.24.1",
+    "eslint-plugin-prettier": "^3.4.1",
+    "eslint-plugin-react": "^7.24.0",
+    "jest": "^27.0.6",
+    "prettier": "^2.3.2"
+  },
+  "peerDependencies": {
+    "@parallelmarkets/vanilla": "workspace:*",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2"
```

Comment:
Would we work with earlier versions of 17 or late versions of 16?

Reply:
Good call! 238ee125e94d411e9ee78490d47e27bff0dcab07